style.css

```diff
@@ -0,0 +1,61 @@
+@import url('/components/css/common.css');
+@import url('/components/css/components.css');
+
+.header-nav {
+    box-shadow: 0px 10px 4px -4px hsl(0deg 0% 0% / 0.2);
+    position: sticky;
+    top:0;
+
+}
+
+.hero {
+   
```

Comment:
Use some code formatter like prettier that will take care of these kinds of spaces.